resources/lib/service.py

```diff
@@ -20,6 +21,24 @@ def __init__(self):
             if self.waitForAbort(10):
                 break
 
+    @staticmethod
+    def onNotification(sender, method, data):  # pylint: disable=invalid-name
+        ''' Handler for notifications '''
+        _kodi = KodiWrapper(None)
+
+        if 'upnextprovider' in sender:
+            import json
+            hexdata = json.loads(data)
+
+            if hexdata:
+                from binascii import unhexlify
+                data = json.loads(unhexlify(hexdata[0]))
+                _kodi.log('[Up Next]: %s, %s, %s' % (sender, method, to_unicode(data)), 'Verbose')
+
+                if 'plugin.video.vrt.nu_play_action' in method:
+                    executebuiltin('PlayerControl(Stop)')
```

Comment:
I think it is preferred to use JSONRPC for this. https://kodi.wiki/view/JSON-RPC_API/v9#Player.Stop
I think you can replace this with:
```python
                jsonrpc(method='Player.Stop')
```

Reply:
Hmm, it does require a `playerid`, which would require this:
```python
def get_playerid():
    ''' Get current playerid '''
    result = dict()
    while not result.get('result'):
        result = utils.jsonrpc(method='Player.GetActivePlayers')
    if not result.get('result'):
        return None
    return result.get('result')[0].get('playerid')
```

Reply:
Done.